script/check_claimed_ideas.pl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env perl
+# check all claimed ideas and make sure they have an IA page.
+# if not then make one
+use FindBin;
+use lib $FindBin::Dir . "/../lib";
+use DDGC;
+use Time::Local;
+my $d = DDGC->new;
+
+BEGIN {
+    $ENV{DDGC_IA_AUTOUPDATES} = 1;
+}
+
+MAIN: {
+    my $claimed_without_page = $d->rs('Idea')
+        ->search({
+            claimed_by          => {'!=' => undef},
+            instant_answer_id   => undef,
+        });
+
+    my @time = localtime(time);
+    my $date = $d->db->format_datetime( DateTime->now );
+
+    while (my $idea = $claimed_without_page->next){
+
+        my $ia = $d->rs('InstantAnswer')
+            ->find(
+                $idea->id,
+                {
+                    result_class =>  'DBIx::Class::ResultClass::HashRefInflator'
```

Comment:
FYI, I'm pretty sure we have the DBIC helper configured to do HashRefInflator (via rs) like:

```perl
$d->rs('InstantAnswer')->search({ id => $idea->id })->hri;
```

Won't work with find, since that returns a Result instance.